rpc/systema/cli2systema.proto

```diff
@@ -23,3 +23,41 @@ message CommandMessageResponse {
   // intercept is the message to give when a user performs an intercept command.
   string intercept = 1;
 }
+
+// copied from systemA
```

Comment:
Why would we ever copy the declaration of an rpc service? Why not use the original?

Reply:
there's precedent for doing this @thallgren -- it's not trivial to import the service as it would require a private repo to be pulled in, which would prevent open source contributors from building telepresence

Reply:
eventually the intercept command can be pushed into a daemon command and at that point the OSS functionality can be separated from the proprietary functionality and we can fix this

Reply:
Ok, that makes sense.